docs/scos/dev/upgradability-services/upgradability-guidelines.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Upgradability guidelines
+description: Reference information for evaluator and upgrader tools.
+last_updated: Nov 25, 2021
+template: concept-topic-template
+---
+
+## Entity name is not unique
+
+The names of the following entities should be unique to the extent of making it impossible to accidentally match the name of a core entity introduced in future:
+
+* Transfers
+* Transfer properties
+* Database tables
+* Database columns
+* Methods
+* Constants
+
+If a minor or major release introduces an entity with the same name, the entity in your project might change behavior or cause issues.
+
+### Making entity names unique
+
+To avoid unexpected issues and achieve the same result, make the names of entities unique. This section contains examples of transfer, database table, and method names. You can apply the same solution to any other entity.
+
+#### Examples of code and related errors
+
+`ProductAbstractStore` transfer name is not unique:
```

Comment:
The content is too big for the list to make sense. The reader does not see the next list item and it may look like a single item. It's ok to use lists for sentences and even short paragraphs, but not for very long content. Take a look at the [intro about lists in the Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation). As suggested in the previous comment, we could convert the items into drop-downs to make it somewhat a list.